begin cleaning pipeline_ace_step.py

craftmine1000 opened the pull request:
begin cleaning pipeline_ace_step.py
fix logger calls in pipeline_ace_step.py
sort + fix + reduce imports in pipeline_ace_step.py

woct0rdho commented:
@ChuxiJ Maybe we can enforce an auto linter like ruff for the codebase and for every PR, to make the codebase cleaner in the long run.

craftmine1000 commented:
since i opened this PR i've cleaned more, done some refactoring, and removed all the built in line-wrapping in pipeline_ace_step.py.

(as there seems to be quite a mix of layout standards around the files i kinda just did my own thing, sorry, i understand if this PR gets dropped)

personally i really like unbounded line lengths while keeping readability, the idea being that you should use a word-wrapping editor that works nicely with python so it fits your screen and gets indented correctly, in my opinion it makes high level readability much better, also makes the code screen agnostic.

some of the multiline structures was setup in those ways for readability and for those i've preserved that by splitting them into multiple statements (q hooking), currently the large function calls are probably the biggest problem, particularly calc_v() and friends given their amount of arguments.

ill update the PR with what i got so far, everything is tested and working.

ChuxiJ commented:
Good suggestion!
